apps/api/scripts/generate-card-pdf.py

```
@@ -309,17 +262,19 @@ def generate_pdf(routine, output_path=None):
     return {
         'layout': 'full',
         'cards_per_page': 1,
+        'pages_generated': copies,
         'pages_generated': 1,
```

Ruff (0.15.7)

[error] Dictionary key literal `'pages_generated'` repeated

(F601)